src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import styled from 'styled-components';
+
+const FooterContainer = styled.footer`
+  display: flex;
+  justify-content: center;
+  width: 100vw;
+  height: auto;
+  background-color: ${({ theme }) => theme.colors.light};
+  padding: ${({ theme }) => theme.sizes.spacing.sm};
+
+  & i {
+    font-size: ${({ theme }) => theme.sizes.fonts.xl};
+  }
+
+  & .footer-i-middle {
+    margin-left: ${({ theme }) => theme.sizes.spacing.sm};
+    margin-right: ${({ theme }) => theme.sizes.spacing.sm};
+  }
```

Comment:
Just put the same margins on all of them. This was good thinking but you'll regret it it 4 months when product asks us to add a new link to snapchat or something

Reply:
Thanks Marks, have changed that